manual/parameterisations.tex

```diff
@@ -328,6 +328,7 @@ \subsubsection{Standard $k-\epsilon$ Turbulence Model}\label{sec:kepsilon}
 \end{equation}
 where $u_z$ is the magnitude of the velocity in the same direction as gravity and $u_{xy}$
 is the magnitude of the velocity in all other directions.
+Multiphase version of the $k$ and $\epsilon$ equations includes a contribution from volume fraction and interphase interaction terms, as discussed in \cite{bhutani2016}. The present multiphase implementation in Fluidity neglects the interphase interaction terms.
```

Comment:
Sorry for the confusion. I meant "The present multiphase implementation of the $k$ and $\epsilon$ equations in Fluidity neglects the interphase interaction terms." This includes the production term due to the fluctuations in the other phases, which are currently not implemented in Fluidity.